frontend/src/views/web/challenge/submission.html

```diff
@@ -12,78 +12,114 @@
                     </div>
                 </div>
                 <div class="ev-md-container ev-panel-title"><strong>Please fill the following fields to submit</strong></div>
-                <div class=" ev-card-body exist-team-card">
-                    <form ng-submit="challenge.makeSubmission()">
-                        <ul>
-                            <b>Select Phase:</b>
-                            <li ng-repeat="item in challenge.phases.results">
-                                <input type="radio" name="selectPhase" class="with-gap selectPhase" id="{{item.id}}" value="{{item.id}}" ng-model="challenge.phaseId">
-                                <label for="{{item.id}}"></label>
-                                <div class="show-member-title pointer"><strong class="text-med-black">Phase: </strong>{{item.name}}
-                                    <br>
-                                    <strong class="text-med-black">End Date: </strong>{{item.end_date | date:'medium'}}</div>
-                                <div class="clearfix"></div>
-                            </li>
+                <div class="col s12 m7">
```

Comment:
@RishabhJain2018 this grid is invalid. There is no row presents and you are calling col. Please add row or make changes accordingly.